docs/modeling/fitting.rst

```diff
@@ -121,7 +122,9 @@ and `~astropy.modeling.functional_models.Trapezoid1D` models and the
     t_init = models.Trapezoid1D(amplitude=1., x_0=0., width=1., slope=0.5,
                                 bounds={"x_0": (-5., 5.)})
     fit_t = fitting.LevMarLSQFitter()
-    t = fit_t(t_init, x, y)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")  # Ignore fit warning.
+        t = fit_t(t_init, x, y)
```

Comment:
One downside of this is that it is a code example people might copy/paste but we don't necessarily want them to copy/paste the ignoring of warnings? Can we choose the data or initial parameters better to avoid a warning in the first place?

Reply:
Maybe a good occasion to show how to change the number of iterations. I see no warning with 
```
t = fit_t(t_init, x, y, maxiter=200)
```

Reply:
Thanks, I applied Simon's suggestion. 🤞